core_tic_tac_toe/game.py

```diff
@@ -0,0 +1,62 @@
+class Game:
+
+    def __init__(self, board, display, game_logic, player_1, player_2):
+        self.board = board
+        self.display = display
+        self.game_logic = game_logic
+        self.player_1 = player_1
+        self.player_2 = player_2
+        self.current_player = self.player_1
+
+    def start(self):
+        while True:
+            self.__show_board()
+
+            self.__prompt_make_move(self.get_current_player())
+
+            moves = self.__get_board()
```

Comment:
`moves` and `__get_board` don't feel like they go together. For example, I'd expect a method named `__get_board` to return a board, not moves. Would it makes sense to rename the local variable, the method, or both?